sql_datafreshness/sql_datafreshness.py

```diff
@@ -141,7 +144,7 @@ def read_opts_file(file_name):
 
     return opts, thresholds
 
-def produce_report(data, general_threshold, adhoc_thresholds, msg_from, msg_to=''):
+def produce_report(args, data, general_threshold, adhoc_thresholds, msg_from, msg_to=''):
```

Comment:
I will need other parameter from the arguments. It will be to create a file with the content of our message. But If you prefer to put two arguments I can do it.